examples/backends/vllm/launch/disagg_kvbm_router.sh

```diff
@@ -10,49 +10,44 @@ export PYTHONHASHSEED=0
 # Common configuration
 MODEL="Qwen/Qwen3-0.6B"
 
-# run decode router with kv-overlap-score-weight 0 for pure load balancing
 python -m dynamo.frontend \
     --router-mode kv \
     --http-port 8000 \
-    --kv-overlap-score-weight 0 \
     --router-reset-states &
 
-# run standalone router service for prefill workers
```

Comment:
@nnshah1 I think back in the days we are using separate routers for disagg, but now we can just have one router.